addons/mass_mailing/static/src/builder/mass_mailing_builder.js

```diff
@@ -30,6 +32,119 @@ export class MassMailingBuilder extends Component {
         toggleFullScreen: { type: Function },
     };
 
+    setup() {
+        const patch = {
```

Comment:
I would make an individual file for this patch to avoid any confusion with `this` usage/context